app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_components = command.split()
+    source_name = command_components[1]
+    destination_name = command_components[2]
+    is_valid_command = command_components[0] == "mv"
+    is_valid_count_of_components = len(command_components) == 3
+
+    if is_valid_command and is_valid_count_of_components:
```

Comment:
if you check for valid command, maybe better to raise some error if not, and don't do all code in one `if`